tests/test-images/Dockerfile.2

```diff
@@ -1,4 +0,0 @@
-FROM scratch
```

Comment:
Ok and no one was using this container?  Does atomic migrate blow up with this situation?  Why shouldn't this work?